safep/plotting.py

```
@@ -281,7 +293,9 @@ def plot_hysteresis(axes,
     xtxt = r'$\lambda$'
     hyst_ax.set_xlabel(xlabel=xtxt, fontsize=fontsize)
 
-    if pdf_type=='KDE':
+    if np.allclose(diff, 0):
+        warn("Hysteresis could not be computed. Do you have both forward and backward data?")
```

Ruff (0.16.0)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)